test/sampling_correlation_matrices_test.cpp

```diff
@@ -38,17 +38,17 @@ MT rebuildMatrix(const VT &xvector, const unsigned int n){
 
 template<typename NT, typename MT>
 Eigen::Matrix<NT, Eigen::Dynamic, 1> getCoefficientsFromMatrix(const MT& mat) {
```

Comment:
Instead of duplicated this function why don't just move in include/sampling/sample_correlation_matrices.hpp ?